modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -1043,6 +1043,9 @@ private String toExampleValueRecursive(String modelName, Schema schema, Object o
             }
             return fullPrefix + example + closeChars;
         } else if (ModelUtils.isArraySchema(schema)) {
+            if (objExample instanceof Iterable) {
```

Comment:
@therve Do you mind adding a 1-liner comment explaining what condition does/fixes?

Reply:
Done, thanks!